### PR TITLE
fix: select dropdown can't be scrolled with keyboard

### PR DESCRIPTION
**Pull request by gitstart**
# Description 
Select dropdown can't be scrolled with keyboard

Fixes  [IssueLink](https://github.com/ParabolInc/parabol/issues/5745)  #5745

## Demo
https://user-images.githubusercontent.com/1501599/202843311-f0dcf82f-881e-4e7e-99f2-31e048897259.mov

## Testing scenarios

- [x] Scenario A
  - Create a team and add more than 7 members to the team then
  - Open the select dropdown and  Interact over the  list items using the keyboard.

## Final checklist

- [x] I checked the [code review guidelines](../docs/codeReview.md)
- [x] I have added [Metrics Representative](../docs/codeReview.md#metrics-representative) as reviewer(s) if my PR invovles metrics/data/analytics related changes
- [x] I have performed a self-review of my code, the same way I'd do it for any other team member
- [x] I have tested all cases I listed in the testing scenarios and I haven't found any issues or regressions
- [ ] Whenever I took a non-obvious choice I added a comment explaining why I did it this way
- [ ] I added the label `One Review Required` if the PR introduces only minor changes, does not contain any architectural changes or does not introduce any new patterns and I think one review is sufficient'
- [x] PR title is human readable and could be used in changelog

### CODE REVIEW

**Comment by gitstart-parabol** [comment]
@BartoszJarocki this PR is ready for review

File: packages/client/components/Menu.tsx [hunk 1]
```diff
@@ -102,17 +104,17 @@ const Menu = forwardRef((props: Props, ref: any) => {
...
       if (idx === null) setActiveIdx(firstIndex)
       else if (menuItemIdxs.includes(idx)) setActiveIdx(idx)
-      else if (idx < firstIndex) setActiveIdx(lastIndex)
-      else if (idx > lastIndex) setActiveIdx(firstIndex)
-      else if (activeIdx && idx > activeIdx) {
-        for (let ii = idx; ii <= lastIndex; ii++) {
+      else if (idx > firstIndex) setActiveIdx(lastIndex)
+      else if (idx < lastIndex) setActiveIdx(firstIndex)
+      else if (activeIdx && idx < activeIdx) {
+        for (let ii = idx; ii >= lastIndex; ii--) {
```
Comment (Dschoordsch) [inline packages/client/components/Menu.tsx:110]: -2 What happened here? Why is this change necessary?

**Review by Dschoordsch — CHANGES_REQUESTED** [review]

**Comment by BartoszJarocki** [comment]
@gitstart what's more, this would be great to have in every PR you submit. it'll save us a lot of review time.

File: packages/client/components/Menu.tsx [hunk 2]
```diff
@@ -157,10 +157,8 @@ const Menu = forwardRef((props: Props, ref: any) => {
   const handleKeyDown = useCallback(
     (e: React.KeyboardEvent) => {
       if (e.key === 'ArrowDown') {
-        e.preventDefault()
         setSafeIdx(activeIdx === null ? null : activeIdx + 1)
       } else if (e.key === 'ArrowUp') {
-        e.preventDefault()
```
Comment (gitstart-instrumental) [inline packages/client/components/Menu.tsx:163]: @BartoszJarocki After some testing, we concluded that the default action to the event `onKeyDown` allows the parent element to automatically scroll to any focusable children elements. Since `e.preventDefault()` was nuked on `ArrowDown` and `ArrowUp`, the default behavior described in the block above was reinstated. This allows for smooth scrolling down and up with the keyboard arrow keys.

Comment (gitstart-parabol) [inline packages/client/components/Menu.tsx:163]: @BartoszJarocki After some testing, we concluded that the default action to the event `onKeyDown` allows the parent element to automatically scroll to any focusable children elements. Since `e.preventDefault()` was nuked on `ArrowDown` and `ArrowUp`, the default behavior described in the block above was reinstated. This allows for smooth scrolling down and up with the keyboard arrow keys.

**Comment by gitstart-parabol** [comment]
Hi @BartoszJarocki @Dschoordsch, this PR is ready for review

File: packages/client/components/Menu.tsx (hunk 2 above)
Comment (Dschoordsch) [reply packages/client/components/Menu.tsx:163]: I couldn't find a reason these are here, so I think you're right in removing these.

**Review by Dschoordsch — APPROVED** [review]
Works, also does not break the emoji dialog
